File: app/src/main/java/com/example/myscope/activities/Full_Image_Activity.java
package com.example.myscope.activities;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;

import com.example.myscope.R;

public class Full_Image_Activity extends BaseActivity {
    ImageView selectedImage;


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.full_image_main);
        selectedImage = (ImageView) findViewById(R.id.selectedImage); // init a ImageView
        Intent intent = getIntent(); // get Intent which was set from adapter of Previous Activity
        selectedImage.setImageResource(intent.getIntExtra("image", 0)); // get image from Intent and set it in ImageView
    }
}

File: app/src/main/java/com/soargtechnologies/myscope/activities/Full_Image_Activity.kt
package com.soargtechnologies.myscope.activities

import android.os.Bundle
import android.view.View
import android.widget.ImageView
import com.soargtechnologies.myscope.R

class Full_Image_Activity : BaseActivity() {
    var selectedImage: ImageView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.full_image_main)
        selectedImage = findViewById<View>(R.id.selectedImage) as ImageView // init a ImageView
        val intent = intent // get Intent which was set from adapter of Previous Activity
        selectedImage!!.setImageResource(intent.getIntExtra("image", 0)) // get image from Intent and set it in ImageView
    }
}